benchmark/assert/deepequal-buffer.js

```diff
@@ -27,6 +27,7 @@ function main(conf) {
   data.copy(expectedWrong);
 
   switch (conf.method) {
+    case '':
```

Comment:
Would you be so kind and a a comment why this is necessary? Otherwise I would be confused by this.

Reply:
Done here and in the other 8 places it occurs in other files.